### PR TITLE
Check in/88841/new landing follow ups

### PR DESCRIPTION
**Pull request by brianseek**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

This PR covers changes for several tickets listed below in the related issues.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#87416
department-of-veterans-affairs/va.gov-team#87422
department-of-veterans-affairs/va.gov-team#87756
department-of-veterans-affairs/va.gov-team#87810

## Testing done

Unit and end to end

## What areas of the site does it impact?

day-of check-in and pre-check-in

## Acceptance criteria
 - [ ] the AC in each listed ticket is covered
 
### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/components/WhatToDoNext.jsx [hunk 1]
```diff
@@ -81,27 +58,49 @@ const WhatToDoNext = props => {
...
-                  <va-card show-shadow={actionableAppointments.length > 1}>
+                  <va-card show-shadow={appointmentCards.length > 1}>
                     <h3
                       className="vads-u-margin-top--0 vads-u-font-size--h4"
                       data-testid="what-next-card-title"
                       id={cardTitleId}
                     >
                       {cardTitle}
                     </h3>
+                    <ul>
```
Comment (wullaski) [inline src/applications/check-in/components/WhatToDoNext.jsx:82]: We discussed showing just a single line without all the list markup here when there is only one item. Is that going to be done in a separate ticket?

File: src/applications/check-in/utils/appointment/index.js [hunk 2]
```diff
@@ -417,7 +417,7 @@ const getCheckinableAppointments = appointments => {
 const convertAppointments = appointments => {
   return appointments.map(appointment => ({
     id: appointment.id,
-    facility: appointment.attributes.location,
+    facility: appointment.attributes.location, // This is going away. We will switch to use facilityName
```
Comment (wullaski) [inline src/applications/check-in/utils/appointment/index.js:420]: Should we just make the change now?

**Review by wullaski — APPROVED** [review]
Looks good just a couple questions.

File: src/applications/check-in/components/WhatToDoNext.jsx (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/components/WhatToDoNext.jsx:82]: there is a ticket for it: https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/89076

Also has a planning poker out for it.

File: src/applications/check-in/utils/appointment/index.js (hunk 2 above)
Comment (brianseek) [reply src/applications/check-in/utils/appointment/index.js:420]: I think we should wait for the hand off from BE because other things might change there.

**Pull request merged**